### ISSUE TITLE
Would be nice to order the ContentPanes in the outline in the order in which they appear in the design

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1037]
I think this amounts to creating the ContentPanes in a particular order to produce an ordering that makes sense to users. But then the question is what order makes sense to users. One could argue that users would want order to go left-to-right, then top-to-bottom, in which case a reasonable ordering might be top, left, center, right, bottom. But another argument is that "center" tends to have the majority of the content, so put it in a predictable place, probably at the end of the list, so maybe top, left, right, bottom, center.

**Comment by peller** [comment]
I think there's some benefit to keeping things simple and matching the order of the markup.  There's already feedback where the user can select individual widgets to see where they are visually.

**Comment by JonFerraiolo** [comment]
Yes, we shouldn't show a different order in Maqetta than what's in the markup. What I'm saying is that when we first create the markup, let's create the new ContentPanes in as intuitive an order as possible.
